apps/test-app/frontend/src/index.tsx

```diff
@@ -23,26 +23,22 @@ Logger.initializeToConsole();
 Logger.setLevelDefault(LogLevel.Warning);
 
 async function initializeApp() {
-  // __PUBLISH_EXTRACT_START__ Presentation.Frontend.RpcInterface.Options
   const iModelAppOpts: IModelAppOptions = {
-    rpcInterfaces,
-  };
-  // __PUBLISH_EXTRACT_END__
-
-  Object.assign(iModelAppOpts, {
     localization: new ITwinLocalization({
       initOptions: { lng: "en" },
     }),
-  });
+  };
 
   if (ProcessDetector.isElectronAppFrontend) {
-    // __PUBLISH_EXTRACT_START__ Presentation.Frontend.IModelAppStartup
     await ElectronApp.startup({ iModelApp: iModelAppOpts });
```

Comment:
Looks like RpcInterfaces are not registered for electron app?

Reply:
yes... fixed